### PR TITLE
Adding example code and correcting sample code in the README.md

### PR DESCRIPTION
**Pull request by domesticmouse**
PTAL @mit-mit

### CODE REVIEW

File: example/main.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import 'package:cassowary/cassowary.dart';
+
+void main() {
+  var solver = Solver();
+  var left = Param(10);
+  var right = Param(20);
+  var widthAtLeast100 = right - left >= cm(100.0);
+  var edgesPositive = (left >= cm(0.0))..priority = Priority.weak;
+  solver
```
Comment (mit-mit) [inline example/main.dart]: I think I would remove the trailing comma in the short list, which gives us better dartfmt output here:

```
  solver
    ..addConstraints([widthAtLeast100, edgesPositive])
    ..flushUpdates();
```

File: README.md [hunk 2]
```diff
@@ -24,28 +24,32 @@ Each expression in turn is made up of a list of `Term`s and a constant. The term
...
-Constraint edgesPositive = (left >= CM(0.0) | Priority.weak)
+var edgesPositive = (left >= cm(0.0))..priority = Priority.weak;
 ```
 
 Once the set of constraints are constructed, they are added to the solver and the results of the solution flushed out.
 
 ```
-solver.addConstraints([widthAtLeast100, edgesPositive])
-     ..flushVariableUpdates();
+solver
```
Comment (mit-mit) [inline README.md]: See below

File: example/main.dart (hunk 1 above)
Comment (domesticmouse) [reply example/main.dart]: Done

File: README.md (hunk 2 above)
Comment (domesticmouse) [reply README.md]: And done.

**Pull request merged**